### ISSUE TITLE
SyntaxError: Expected ',' or ']' after array element in extensions.json JSON

### ISSUE THREAD

**Issue by paulober** [raspberrypi/pico-project-generator#81]
The `update_vscode_json_to_sdk1_5` branch contains some changes to to the recommended extensions but a missing causes `JSON.parse` in nodejs to fail.
That causes VS Code to skip this file => no extension gets recommended.

**Comment by lurch** [comment]
Fixed by #82 , so can be closed?

**Comment by paulober** [comment]
@lurch Yes. Github just waits for merging into master to do it automatically.
